src/main/kotlin/de/danielmaile/mpp/item/ToolTier.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * This file is part of MPP.
+ * Copyright (c) 2023 by it's authors. All rights reserved.
+ * MPP is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * MPP is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with MPP.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package de.danielmaile.mpp.item
+
+// Copied minecraft default Tiers.java at net.minecraft.worlds.item.Tiers
+enum class ToolTier(
+    val miningLevel: Int,
+    val itemDurability: Int,
+    val miningSpeed: Float,
+    val attackDamage: Float, // Minecraft uses this and another value on top, do they sum or what?
```

Comment:
We should find out what we need and what we don't then comment what it does instead of a question.

Reply:
We left blocks and implicitly tools for another branch, I actually managed to get what happens with the value but didn't work on it anymore after that. However, we can remove the comment.
```suggestion
    val attackDamage: Float,
```